pip/req/req_install.py

```diff
@@ -41,6 +41,26 @@
 from pip._vendor.packaging.version import Version
 
 
+FILTER_INSTALL_OUTPUT_REGEX = re.compile(r"""
+    ^running .* |
+    ^writing .* |
```

Comment:
Because of ```re.VERBOSE``` the space after the word isn't treated as a character to match any more. It's not a big deal, but maybe ```^running\s.*``` (etc) would be better. Similarly for a number of the others below.